src/api/app/components/bs_request_history_element_component.rb

```diff
@@ -27,15 +27,6 @@ def icon
     end
   end
 
-  # While all history elements possibly have a comment, not all of them are from an actual human...
-  def element_with_comment_from_human?
-    ['RequestReviewAdded', 'ReviewAccepted', 'ReviewDeclined', 'RequestAccepted', 'RequestDeclined'].include?(@element.type.demodulize)
```

Comment:
While for the most of the types it is fine to hide the message behind an icon and have it displayed only on hover with the tooltip, maybe it would still be important to have it always visible in case it is a `RequestAccepted` and `RequestDeclined`. For all  the other cases I'm all for it to hide it, but for the last two I'd still keep them. What do you think?

Reply:
I agree, they are the only ones that might be relevant. Especially the Decline, which might contain some relevant reasoning.

I'll try something else for them

Reply:
We can consider these three as "final states":

- RequestAccepted
- RequestRevoked
- RequestDeclined

But the reason for a re-opening might be relevant as well... :thinking: 

Too many exceptions maybe? I better go for all collapsed and the hover reveals the reason/comment.

I also think an info icon has a more clear meaning than a file. Please compare:

![Screenshot 2023-11-21 at 14-22-43 Open Build Service](https://github.com/openSUSE/open-build-service/assets/2581944/7c1dcef4-74a1-4213-94c5-20372c9a6c15)

Reply:
Finally, we are going with collapse.

![Screenshot 2023-11-21 at 17-31-17 Open Build Service](https://github.com/openSUSE/open-build-service/assets/2581944/874eac04-3b83-48b6-b39a-1c019aba516e)